### PR TITLE
WP Super Cache: Add Easy settings tab

### PR DESCRIPTION
**Pull request by donnapep**
This PR adds the settings UI for the _Easy_ tab. The _Recommended Links and Plugins_ section was not ported from the original plugin.

### Single Site

easy-single-site

### Multi-Site

easy-multi-site

### CODE REVIEW

File: client/extensions/wp-super-cache/components/easy.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+		const { checkedRadio } = this.state;
+
+		return (
+			<div>
+				<SectionHeader label={ translate( 'Caching' ) }>
+					<Button
+						compact={ true }
+						primary={ true }
+						type="submit">
+							{ translate( 'Update Status' ) }
```
Comment (a8ci18n) [inline client/extensions/wp-super-cache/components/easy.jsx]: Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'Status Update' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/wpsc-easy-settings)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by donnchawp** [comment]
The preview looks great. However, there is one form on that page which is missing. On multisite installs there's a "Delete Cache On All Blogs" button at the end. Should I add an API endpoint that returns that status?

**Comment by donnapep** [comment]
@donnchawp Hmmm...not too sure. I believe I can get whether or not a site is multisite, but I'll need to set up a multisite install to be sure. For now, you can assume I don't need an endpoint for this. I'll be sure to let you know if I learn otherwise. Thx!

**Comment by donnapep** [comment]
@donnchawp Would you mind giving the last commit a review? It adds the site to the URL (e.g. `extensions/wp-super-cache/donnapep.wordpress.com`. If you browse directly to `extensions/wp-super-cache`, you'll be presented with the site selector. Using the site information, we can check that the site is a multi-site Jetpack install, and show the additional button in that case.

Once I get a 👍 from you, I think we'll be good to deploy this one.

Thx!

**Comment by donnchawp** [comment]
That looks great!

File: client/extensions/wp-super-cache/constants.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const Tabs = {
```
Comment (dbtlr) [inline client/extensions/wp-super-cache/constants.js]: A little late, but I forgot to submit these comments I made before. :)

**Pull request merged**